### PR TITLE
Log levels

### PR DESCRIPTION
**Pull request by klautcomputing**
This introduces the concept of log levels, deletes a lot of duplicate tests, refactors a lot of code and introduces 

```
-v or --verbose to set the log level
--verbose=ERROR
--verbose=WARNING
--verbose=INFO
--verbose=DEBUG
```

The default log level is INFO and it will print INFO, WARNING and ERROR

Supported log levels are `info`, `warn` and `error` (for now, but more can be introduced easily if needed)

@marc-barry to confirm that what was meant to work in #13 actually works
@JonPulsifer to make sure all the features do what they should do (especially privilege needs a look and potentially some work)
@jinankjain and @csgregorian for reviewing my code

### CODE REVIEW

File: cmd/util.go [hunk 1]
```diff
@@ -82,17 +83,59 @@ type kubeAuditReplicationControllers struct {
...
-	name        string
-	capsAdded   []apiv1.Capability
-	imgName     string
-	capsDropped bool
-	kubeType    string
-	dsa         string
-	sa          string
-	token       *bool
-	imgTag      string
+	Err         int
```
Comment (jinankjain) [inline cmd/util.go]: Is there any specific reason that you want to make these variable public instead of private?

Comment (klautcomputing) [reply cmd/util.go]: Yeah, afaik reflect only works on public variables, so to make the meta programming part work (that prints the correct fields of the `results` array) they needed to be exported.

**Comment by jinankjain** [comment]
Could you please also  remove the test yml for other resources?

**Comment by klautcomputing** [comment]
I will do that in one of my next PRs when I introduce a test framework, unless you feel strongly about doing it in this PR.

**Pull request merged**